From ecf98a5629d252b6fb2c9aa5baa9282828f3243b Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 14 Oct 2013 13:35:21 +0200
Subject: [PATCH 14/18] qemu-char: report udp backend errors

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1381757723-23134-14-git-send-email-kraxel@redhat.com>
Patchwork-id: 54855
O-Subject: [RHEL-7 qemu-kvm PATCH 13/15] qemu-char: report udp backend errors
Bugzilla: 922010
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit 58a3714c529b1b97ee078ea25b58f731aabcbfff)
---
 qemu-char.c | 2 ++
 1 file changed, 2 insertions(+)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 qemu-char.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/qemu-char.c b/qemu-char.c
index fc1b21a..bacc7b7 100644
--- a/qemu-char.c
+++ b/qemu-char.c
@@ -2280,6 +2280,8 @@ static CharDriverState *qemu_chr_open_udp(QemuOpts *opts)
 
     fd = inet_dgram_opts(opts, &local_err);
     if (fd < 0) {
+        qerror_report_err(local_err);
+        error_free(local_err);
         return NULL;
     }
     return qemu_chr_open_udp_fd(fd);
-- 
1.7.1

